api/dev/setup-dev-user-in-container.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env sh
```

Comment:
We don't need this approach anymore, check the other projects... `run-as-user.sh`

Reply:
The "run-as-user" approach doesnt work. For some reason groupmod returns an error, which means that we need to use groupadd, which means that we need to check if the group exists before calling groupadd, which means that we will end with something similar to the "setup-dev-user"